content/administration/install/cdn.rst

```diff
@@ -1,80 +1,109 @@
-===========================================
-Setting up a Content Delivery Network (CDN)
-===========================================
+=======================================
+Set up a content delivery network (CDN)
+=======================================
 
 .. _reference/cdn/keycdn:
 
-Deploying with KeyCDN_
-======================
+Deploying with KeyCDN
+=====================
 
-.. sectionauthor:: Fabien Meghazi
+A :abbr:`CDN (Content Delivery Network)` or *content distribution network*, is a geographically
+distributed network of servers that provides high speed internet content. The :abbr:`CDN (Content
+Delivery Network)` provides quick, high-quality content delivery for content-heavy websites.
 
-This document will guide you through the setup of a KeyCDN_ account with your
-Odoo powered website.
+This document will guide you through the setup of a KeyCDN_ account with an Odoo powered website.
 
 Step 1: Create a pull zone in the KeyCDN dashboard
 --------------------------------------------------
```

Comment:
Don't need `Step #` labels in the headings since we indicate that this is a first step in the body copy.
```suggestion
Create a pull zone in the KeyCDN dashboard
------------------------------------------
```